pkg/restore/log_client.go

```diff
@@ -298,11 +298,6 @@ func (l *LogClient) collectRowChangeFiles(ctx context.Context) (map[int64][]stri
 			SubDir:    dir,
 			ListCount: -1,
 		}
-		if ok, err := l.restoreClient.storage.FileExists(ctx, opt.SubDir); !ok {
-			// this could happen after duplicate create drop table
-			log.Debug("path not exists", zap.String("path", opt.SubDir), zap.Error(err))
-			continue
-		}
```

Comment:
Why remove it? Is it a bug?

Reply:
Because it unnecessary. since `WalkDir` can handle not exists directory, And this `FileExists` has different meaning in S3 storage and local storage. So I removed it